test/xapi/track_test.rb

```diff
@@ -68,32 +65,32 @@ def test_doc_format
 
   def test_track_with_gitter_room
     track = Xapi::Track.new('fruit', FIXTURE_PATH)
-
-    assert_equal "xfruit", track.gitter
+    assert_equal 'xfruit', track.gitter
   end
 
   def test_track_with_default_checklist_issue
     track = Xapi::Track.new('fruit', FIXTURE_PATH)
-
     assert_equal 1, track.checklist_issue
   end
 
   def test_custom_test_pattern
     track = Xapi::Track.new('fruit', FIXTURE_PATH)
-
-    files = [
-      "file.ext",
-      "test.ext",
-      "file.tst",
-      "sub/dir/file.ext",
-      "sub/dir/file.tst",
-    ]
-    assert_equal %w(file.tst sub/dir/file.tst), files.select {|f|
-      f =~ track.test_pattern
-    }
+    assert_equal(/\.tst$/, track.test_pattern)
   end
 
   def test_unknown_track
     refute Xapi::Track.new('nope', FIXTURE_PATH).exists?, "unexpected track 'nope'"
   end
+
+  def test_icon_url
+    subject = Xapi::Track.new('fake', FIXTURE_PATH)
+    expected = 'http://x.exercism.io/v3/tracks/fake/img/icon.png'
+    assert_equal expected, subject.icon_url
+  end
+
+  def test_icon_url_nonexisting
+    subject = Xapi::Track.new('noicon', FIXTURE_PATH)
+    expected = nil
```

Comment:
if 'this' is 'test_icon_url_nonexisting'
It exercises an uncovered code branch. Does it need a better name? I didn't include it with `test_icon_url` because that is testing different behaviour.

if you're asking about 'expected' it's because it follows the three line 'subject/expectation/assertion' pattern. 
I'd rather assert that something that I'm usually expecting to be a string is nil than use refute.